src/FullScreenWindow.vala

```diff
@@ -135,23 +135,18 @@ public class FullscreenWindow : PageWindow {
         switch (Gdk.keyval_name (event.keyval)) {
             case "F11":
             case "Escape":
-                on_close ();
+                close ();
                 return true;
         }
 
         return (base.key_press_event != null) ? base.key_press_event (event) : false;
     }
 
-    private void on_close () {
+    public new void close () {
```

Comment:
I would avoid redefining a `close` function but use another name instead

Reply:
I guess thinking about this there are probably other calls to `close` in subclasses that we don't know about so maybe better to not touch it :grimacing: